src/Microsoft.Identity.Client/Instance/Discovery/IKnownMetadataProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+
```

Comment:
should add space here

Reply:
Will do. Let me know if you want to chat so I can provide more details if you want to do a more in depth review.

Reply:
Lets do that. I have an idea of what you are doing here but I dont understand the full context yet

Reply:
maybe better to set something up w/all of us, so you just do it once?

---
In reply to: [298253038](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1229#discussion_r298253038) [](ancestors = 298253038)